src/plugins/discover/public/context_awareness/profile_providers/logs_data_source_profile/accessors/index.ts

```diff
@@ -7,3 +7,4 @@
  */
 
 export { getRowIndicatorProvider } from './get_row_indicator_provider';
+export { getCellRenderers } from './get_cell_renderers';
```

Comment:
Would it be possible to use the same renders also for the badge in Log Overview tab?

<img width="1407" alt="Screenshot 2024-07-16 at 16 23 49" src="https://github.com/user-attachments/assets/d36b99ae-91bd-4cbe-af60-7bb5e773c477">

Reply:
Definitely doable, although I'd prefer to save it for a followup if we do since it'll require some refactoring of the current log level chip component. @elastic/obs-ux-logs-team WDYT about this suggestion to update the log level chip to use the new log badge component for consistency? If we all agree, I'll create a separate issue to implement this as a followup.

Reply:
Always in favour of visual consistency, +1 for updating it and make them look the same 👌

Reply:
Great! I created a followup issue to take care of it here: #188553.